install_deps.sh

```diff
@@ -1,15 +1,14 @@
 #!/bin/sh
+
+set -ex
+
 apt-get update
 
-apt-get install vim imagemagick optipng --yes
+apt-get install ghostscript imagemagick optipng --yes
+
+rm -rf /var/lib/apt/lists/*
 
-wget -q https://s3.amazonaws.com/sl-public-dev-assets/ghostscript-9.15.tar.gz -O /tmp/ghostscript-9.15.tar.gz
```

Comment:
IDK what the custom build of Ghostscript was for, or whether it's still necessary. I *suspect* that we needed a newer version that didn't have a Debian pakage, but I don't know for sure. cc @ShaneKilkelly @henryoswald

Reply:
FWIW I'm not aware of any need for a custom build. I am fairly certain that it is not still necessary in the SaaS edition, because the feature that used ghostscript is presently disabled. (If you uploaded a .pdf to your project, the filestore would use ghostscript to generate a preview as a PNG/JPG that would work in-browser. We disabled it due to security concerns, but it's still available as an option in CE/SP builds.)

Reply:
The [templates service uses that preview feature too](https://github.com/overleaf/templates-api/blob/14bd491f5fc9c05f1de8a6a855dfa0f8378e2492/app/coffee/filestore/TemplateBuilder.coffee#L96-L97), but I am not sure if its still in use.

Reply:
That is a good point. It's no longer used in SaaS (we're not publishing any more SL templates) but again I think it would be used in SP.

Reply:
cc @mserranom

Reply:
no `install_deps.sh` is run in CE/ServerPro, this change shouldn't have any effect (and `ghostscript` is not installed in the image)

Reply:
We had a custom build because ghostscript latest release at the time wasn't very up to date and head had some args we needed to use. If I remember correctly the Ghostscript dev's actually added them in for us very quickly.